src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
-// write code here
+const list = document.querySelectorAll('li');
+const personList = document.querySelector('ul');
+
+function getNum(string) {
+  return +string.replace(/[$,]/g, '');
+}
+
+function sortList(li) {
+  const sortPerson = [...li].sort((a, b) =>
+    getNum(b.dataset.salary) - getNum(a.dataset.salary));
+
+  personList.append(...sortPerson);
+}
+
+function getEmployees(li) {
+  const employeesArr = [...li].map(person => {
```

Comment:
no needs to create copy of array for `map`
```suggestion
  const employeesArr = li.map(person => {
```